gateway-js/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 > The changes noted within this `vNEXT` section have not been released yet.  New PRs and commits which introduce changes should include an entry in this `vNEXT` section as part of their development.  When a release is being prepared, a new header will be (manually) created below and the appropriate changes within that release will be moved into the new section.
 
--_Nothing yet! Stay tuned_.
+- OpenTelemetry will now include the GraphQL `operationName` in span attributes, following up on the initial implementation introduced in v0.31.0 via [#836](https://github.com/apollographql/federation/pull/836)
```

Comment:
@lennyburdette I've added this `CHANGELOG.md` if you want to give it a quick once-over.

Reply:
perfect, thank you!